LedRGB.cpp

```diff
@@ -9,21 +9,15 @@ LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
 }
 
 void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
-    r /= 2.55;
-    g /= 2.55;
-    b /= 2.55;
-    if (checkBrightness) {
-        RedLed.setBrightness(r);
-        GreenLed.setBrightness(g);
-        BlueLed.setBrightness(b);
-    }
+    setBrightness(r);
+    setBrightness(g);
+    setBrightness(b);
```

Comment:
Ehh,yes and no color should be independent of brightness.

`LedRGB myRGB;`
`myRGB.setRGB(255,255,50);`
`myRGB.setBrightness(50);`

Red led is 50%, Green is 50%, Blue is 9.8%

`myRGB.setBrightness(50);`
`myRGB.setBrightness(50);`
`myRGB.setBrightness(50);`

Red led is still 50%, Green is still 50%, Blue is still 9.8%